src/cider/nrepl/middleware/util/nrep.clj

```diff
@@ -0,0 +1,26 @@
+(ns cider.nrepl.middleware.util.nrepl
+  "Common utilities for interaction with the client."
+  (:require
+   [clojure.tools.nrepl.middleware.interruptible-eval :refer [*msg*]]
+   [clojure.tools.nrepl.misc :refer [response-for]]
+   [clojure.tools.nrepl.transport :as transport]))
+
+
+(defn send-user-message
```

Comment:
Maybe those two functions can be folded into one with one extra param (the type of the message)?